src/Plugin/Guard/AuthorizationService.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Prooph\ServiceBus\Plugin\Guard;
+
+/**
+ * Interface AuthorizationService
+ * @package Prooph\ServiceBus\Plugin\Guard
+ */
+interface AuthorizationService
+{
+    /**
+     * Check if the permission is granted to the current identity
+     *
+     * @param string $permission
+     * @param mixed  $context
+     * @return bool
+     */
+    public function isGranted($permission, $context = null);
```

Comment:
$permission is in our case always the `message name`, so we should name the property `$messageName` to make this explicit.